giskard-server/src/main/resources/aitest/code_test_templates/data_drift.yml

```diff
@@ -54,23 +61,29 @@ items:
         and that we cannot assume drift for this variable.
 
         Args:
-            actual_series(GiskardDataset):
-                Categorical column in actual dataset
-            reference_series(GiskardDataset):
-                Categorical column in reference dataset
-            threshold:
+            actual_ds(GiskardDataset):
+                Actual dataset to compute the test
+            reference_ds(GiskardDataset):
+                Reference dataset to compute the test
+            column_name(str):
+                Name of column with categorical feature
+            threshold(float):
                 Threshold for p-value of chi-square
 
         Returns:
+            actual_slices_size:
+                Length of rows with given categorical feature in actual slice
+            reference_slices_size:
+                Length of rows with given categorical feature in reference slice
             metric:
                 The pvalue of chi square test
-
             passed:
                 TRUE if metric > threshold
       """
       tests.drift.test_drift_chi_square(
-          actual_series=actual_ds.df['{{CATEGORICAL FEATURE NAME}}'],
-          reference_series=reference_ds.df['{{CATEGORICAL FEATURE NAME}}'],
+          actual_ds=actual_ds.df,
```

Comment:
Now that test requires a GiskardDataset as an input the examples in yaml files need to be adapted, so here for example it needs to be actual_ds instead of actual_ds.df

Reply:
Lets discuss this tomorrow to get a better understanding